Lesson6/src/test/java/MainTest.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MainTest {
+    Main main = new Main();
+
+    @Test(expected = RuntimeException.class)
+    public void testArrayAfterFourException(){
+        int [] arr = {1,3,2,3};
+        int [] expectedArr = {};
+        int [] result = main.arrayAfterFour(arr);
+        Assert.assertArrayEquals(expectedArr, result);
+    }
+
+    @Test
+    public void TestArrayAfterFour(){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.